store/block_store.go

```diff
@@ -150,6 +156,27 @@ func (s *TendermintBlockStore) GetTxResult(txHash []byte) (*ctypes.ResultTx, err
 	}, nil
 }
 
+func (s *TendermintBlockStore) GetTxResultByHeightAndIndex(height *int64, index int) (*ctypes.ResultTx, error) {
```

Comment:
Adding this to the block store seems premature

Reply:
Removed